datadog_checks_dev/datadog_checks/dev/tooling/commands/release/testable.py

```diff
@@ -156,9 +156,10 @@ def testable(ctx, start_id, agent_version, milestone, dry_run):
             '2': 'Containers',
             '3': 'Logs',
             '4': 'Platform',
-            '5': 'Process',
-            '6': 'Trace',
-            '7': 'Integrations',
+            '5': 'Networks',
+            '6': 'Processes',
```

Comment:
It seems this change only applies when on the `datadog-agent` repo.

Are there situations when we'd expect changes in `integrations-core` to be assignable to them? (Or to Logs or APM, since those aren't listed in the `options` for `integrations-core` either…)

If so, then maybe we could move things around a bit…

```python
teams = ['Containers', 'Logs', 'Platform', 'Networks', 'Processes', 'Trace']

if repo == 'integrations-core':
    teams = ['Integrations', 'Core'] + teams
else:
    teams = ['Core', 'Integrations'] + teams

options = {str(index + 1): team for index, team in enumerate(teams)}
options['s'] = 'Skip'
options['q'] = 'Quit'
```

Reply:
I think it was intentional: all these teams weren't initially in the `integrations-core` list because we don't expect them to make modifications here (only Core, Platform, Integrations, Containers & Docs own things in this repo).
We could add them just in case, it depends on what your team wants (since you'll be the ones using it on this repo).

Reply:
Yup intentional

Reply:
OK, sounds good! Let's leave this like that then.